providers/class-two-factor-totp.php

```diff
@@ -78,6 +79,21 @@ public function get_label() {
 		return _x( 'Time Based One-Time Password (TOTP)', 'Provider Label', 'two-factor' );
 	}
 
+	/**
+	 * Enqueue scripts
+	 */
+	public function enqueue_assets() {
+		$environment_prefix = file_exists( TWO_FACTOR_DIR . '/dist' ) ? '/dist' : '';
+
+		wp_enqueue_script(
+			'two-factor-qr-code-generator',
+			plugins_url( $environment_prefix . '/includes/qrcode-generator/qrcode.js', __DIR__ ),
+			array(),
+			TWO_FACTOR_VERSION,
+			true
+		);
```

Comment:
As this is being loaded on all admin pageviews based on the hook in use, this should be a `wp_register_script()` and `wp_enqueue_script()` called during the render below.